src/Login.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { useState, ChangeEvent, FormEvent } from 'react';
+import {
+  Text,
+  Button,
+  TextInput,
+  PasswordInput,
+} from '@mantine/core'
+
+interface FormValues {
+  password: string;
+  email: string;
+  confirmPassword?: string;
+}
+
+export default function Login() {
+  const [isRegistered, setIsRegistered] = useState(true)
+  const [formValues, setFormValues] = useState<FormValues>({email: '', password: '', confirmPassword: ''});
+  
+  const handleChangeIsRegistered = (): void => {
+    setIsRegistered(!isRegistered)
+  }
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const {name, value} = event.target;
+    setFormValues((prev) => ({...prev, [name]: value}));
+  }
+
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    console.log('email: ', formValues.email);
+    console.log('password: ', formValues.password);
+  }
+
+  return (
+    <div>
+      {isRegistered === true ? 
+      <>
+        <Text>Log In</Text>
+        <form onSubmit={handleSubmit}>
+          <TextInput
+            label="Email"
+            name='email'
+            placeholder="Enter your email"
+            value={formValues.email}
+            onChange={handleChange}
+            required
+          />
+          <PasswordInput
+            label="Password"
+            name='password'
+            placeholder="Enter your password"
+            value={formValues.password}
+            onChange={handleChange}
+            required
+          />
+          <Button onClick={handleChangeIsRegistered}>Not yet registered?</Button>
```

Comment:
these buttons aren't appearing correctly on the page, the conflict between mantine and tailwind hasn't been resolved yet. As a hint you need to update a setting in the tailwind config

Reply:
hmm, looking through the tailwind docs, the options looked like using either the important: true or prefix: 'tw-', and having tried both, I was able to get the tw- flag to work, but it seems to only add extra work for me as I then had to refactor so that all my tw classnames had tw in front of them and the element just had no styling at all and was only visible on hover when I first made the change. I think I am missing something here but after spending about an hour and a half playing with it and looking at different things, I am ready to ask for additional help with this portion

Reply:
So the answer is actually tailwind has a preflight step in it's initialization that wipes out all existing HTML default styling. Mantine relies on some of this styling to function. To make the two work, you just need to disable tailwind preflight